website/docs/src/content/cli-reference/run.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: gensx run
+description: Execute a deployed workflow on GenSX Cloud
+---
+
+# gensx run
+
+The `gensx run` command executes a workflow that has been deployed to GenSX Cloud.
```

Comment:
Should we say somewhere that the command infers your project from the current directory?